dev-libs/libbpf/libbpf-9999.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs git-r3
+
+DESCRIPTION="Stand-alone build of libbpf from the Linux kernel"
```

Comment:
What about HOMEPAGE?
HOMEPAGE="https://git.kernel.org/pub/scm/linux/kernel/git/bpf/bpf-next.git"

Reply:
Homepage added, thank you for advice.